### ISSUE TITLE
Metrics for rule-loader

### ISSUE THREAD

**Issue by stvnwrgs** [Nordstrom/prometheusRuleLoader#6]
It would be great to have some metrics for the rule-loader itself.

Like for example a counter for rejected rules that can be alerted on.

**Comment by SleepyBrett** [comment]
Good idea! I just was rewriting this for use with prometheus2 against client-go. I'll add that as a stretch goal.

I didn't find a good way to verify rules in 2.x, I just try to unmarshal the yaml and if it passes great, if not it just doesnt get added. But I'd still like a little counter for failed configmap keys and maybe push events back onto the configmaps.

Sorry I haven't been paying attention to the repo!

**Comment by SleepyBrett** [comment]
I should restate, I could only find a way to verify rule groups not individual rules, therefore I'm currently skipping groups that fail validation.
